valhalla/baldr/graphtileheader.h

```diff
@@ -687,6 +711,9 @@ class GraphTileHeader {
   // GraphTile data size in bytes
   uint32_t tile_size_;
 
+  // Offset to the beggining of the OSM ids for nodes data.
+  uint32_t osmids_for_nodes_offset_;
```

Comment:
I guess one could get this from difference between the lane connectivity offset from the end of tile offset and save yourself a slot, but since you can only do this for some subset of offsets, that feels kinda hacky.

Reply:
nah this is all good. if we ever run out of slots we can always get creative and add a secondary header or something haha. but since it wouldnt be at the beginning of the tile we'll have to call it a middler or something haha